scripts/build.sh

```diff
@@ -69,7 +69,7 @@ else
    EXT="sh";
 fi
 # This line will break if there is more than one installer in the folder.
-INSTALLER_PATH=$(find . -name "M*forge*.${EXT}" | head -n 1)
+INSTALLER_PATH=$(find . -iname "M*forge*.${EXT}" | head -n 1)
```

Comment:
Why this change?

Reply:
I was debugging a [Windows issue with `find` not getting anything](https://github.com/conda-forge/miniforge/actions/runs/5542141642/jobs/10116439026#step:4:1933). I tried that one and it's been working ever since, but it could also be a one-off thing (maybe a race condition with the pipes?).

I'll see if removing it has any effect or not.